sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -604,8 +604,10 @@ export interface Location {
     unavailable?: boolean;
 }
 
+export { Location_2 as Location }
```

Comment:
I don't understand why this change is occurring. Can anyone share some context?

Reply:
seems more like a api-extractor bug to me

Reply:
If that's the case, let's revert to an older api-extractor version for the time being.

Reply:
The version on my machine must still be the old one, ran `rush build` and pushed the result to the branch for this PR. The net result is 0 changes.

Reply:
Can you try running on latest master?

Reply:
After running `rush update`, I end up in the same state as you. Looks like the issue is with the latest version of api-extractor. Version 7.7.11 is fine, but 7.7.12 is not.

#8178 updated the version we use to 7.7.12

Can we pin the version we use to 7.7.11 until a fix is shipped?

Reply:
Yes I can send a PR to pin to 7.7.11 for api-extractor.
Should we also open a bug in api-extractor repo for this?

Reply:
We should log a bug, but after coming up with a smaller repro example, otherwise we wont have useful information to share other than saying that the result is different now.

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/8400